jsonnet/kube-prometheus/addons/ksm-lite.libsonnet

```diff
@@ -16,8 +16,8 @@ local addArgs(args, name, containers) = std.map(
             containers: addArgs(
               [|||
                 --metric-denylist=
-                kube_*_created,
-                kube_*_metadata_resource_version,
+                kube_.+_created,
+                kube_.+_metadata_resource_version,
```

Comment:
[Those are already on `main` branch](https://github.com/prometheus-operator/kube-prometheus/blob/main/jsonnet/kube-prometheus/addons/ksm-lite.libsonnet#L19-L20) and shouldn't be added again.

Reply:
Removed this from my branch